File: javascript-closure-angular-client/API/Client/AdditionalInformation.js
goog.provide('API.Client.AdditionalInformation');

/**
 * @record
 */
API.Client.AdditionalInformation = function() {}

/**
 * Additional information of fiscal interest
 * @type {!string}
 * @export
 */
API.Client.AdditionalInformation.prototype.fiscalInfo;

/**
 * Additional information of user interest
 * @type {!string}
 * @export
 */
API.Client.AdditionalInformation.prototype.complementaryInfo;

/**
 * Referenced Process
 * @type {!Array<!API.Client.AdditionalInformation_procRef>}
 * @export
 */
API.Client.AdditionalInformation.prototype.procRef;

File: async-scala-client/src/main/scala/io/swagger/client/model/AdditionalInformation.scala
package io.swagger.client.model

import org.joda.time.DateTime
import java.util.UUID


case class AdditionalInformation (
  fiscalInfo: Option[String],  // Additional information of fiscal interest
complementaryInfo: Option[String],  // Additional information of user interest
procRef: Option[List[AdditionalInformation_procRef]]  // Referenced Process
)
